zkevm-circuits/src/evm_circuit.rs

```diff
@@ -413,11 +556,6 @@ impl<F: Field> Circuit<F> for EvmCircuit<F> {
             block.circuits_params.max_calldata,
         )?;
         block.rws.check_rw_counter_sanity();
-        config.rw_table.load(
```

Comment:
This was moved to `synthesis_sub` since now evm_circuit as `Circuit`/`SubCircuit` are needed to load rw_table (because it's separated from state_circuit already)